packages/DatePicker/src/DatePicker.js

```diff
@@ -241,13 +241,14 @@ function DatePicker(props) {
           ref={calendarRef}
           shouldShowCalendar={shouldShowCalendar}
         >
-          <Calendar
-            date={date}
-            isVisible={shouldShowCalendar}
-            onSelect={handleSelect}
-            possibleDate={debouncedPossibleDate}
-            resetPossibleDate={handleResetPossibleDate}
-          />
+          {shouldShowCalendar ? (
```

Comment:
Unless i'm missing something it seems we no longer need to keep the calendar rendered (but invisible) in the dom after the recent popup changes - https://github.com/acl-services/paprika/pull/249

@allison-c Can you check Is there some functionality that this is meant to be doing now i have removed this complexity?